packages/vaadin-lumo-styles/src/mixins/base-layer-reset.css

```
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+@layer base {
+  :is(:host, *, #id) {
+    &, &::before, &::after {
```

nit: Prettier transformed it to this

```suggestion
    &,
    &::before,
    &::after {
      all: revert-layer;
    }
```